plugin/pkg/admission/storage/persistentvolume/label/admission.go

```diff
@@ -117,20 +118,26 @@ func (l *persistentVolumeLabel) Admit(ctx context.Context, a admission.Attribute
 		return admission.NewForbidden(a, err)
 	}
 
-	requirements := make([]api.NodeSelectorRequirement, 0)
 	if len(volumeLabels) != 0 {
 		if volume.Labels == nil {
 			volume.Labels = make(map[string]string)
 		}
+
+		deprecatedTopologyReqs := make([]api.NodeSelectorRequirement, 0)
+		topologyReqs := make([]api.NodeSelectorRequirement, 0)
 		for k, v := range volumeLabels {
 			// We (silently) replace labels if they are provided.
 			// This should be OK because they are in the kubernetes.io namespace
 			// i.e. we own them
 			volume.Labels[k] = v
 
 			// Set NodeSelectorRequirements based on the labels
+			//
+			// we currently set both beta (failure-domain.beta.kubernetes.io/zone) and
+			// GA (topology.kubernetes.io/zone) topology labels for volumes but only set beta labels
```

Comment:
are we avoiding setting GA labels for `NodeAffinity` using the Beta node labels?

Reply:
nope, this is stale comment, will update shortly